Infodev doc review (#944)
IntelPython/dpctl: README.md

[![Coverage Status](https://coveralls.io/repos/github/IntelPython/dpctl/badge.svg?branch=master)](https://coveralls.io/github/IntelPython/dpctl?branch=master)
![Generate Documentation](https://github.com/IntelPython/dpctl/actions/workflows/generate-docs.yml/badge.svg?branch=master)

<img align="left" src="https://spec.oneapi.io/oneapi-logo-white-scaled.jpg" alt="oneAPI logo" width="75"/>

# Data Parallel Control

> Data Parallel Control or `dpctl` is a Python* library that allows users

Reviewer: ```suggestion
Data Parallel Control or `dpctl` is a Python\* library that allows users
```

to *control* the execution placement of a [compute
kernel](https://en.wikipedia.org/wiki/Compute_kernel) on an
[XPU*](https://www.intel.com/content/www/us/en/newsroom/news/xpu-vision-oneapi-server-gpu.html).

The compute kernel can be a code:
* written by the user, e.g., using `numba-dppy`
* that is part of a library, such as oneMKL  

> The `dpctl` library is built upon the [SYCL*

Reviewer: ```suggestion
The `dpctl` library is built upon the [SYCL\*
```

standard](https://www.khronos.org/sycl/). It also implements Python
bindings for a subset of the standard [runtime
classes](https://www.khronos.org/registry/SYCL/specs/sycl-2020/html/sycl-2020.html#_sycl_runtime_classes).
Such classes allow users to:
* query platforms
* discover and represent devices and sub-devices
* construct contexts and queues 

> `dpctl` features classes for [SYCL* Unified Shared Memory

Reviewer: ```suggestion
`dpctl` features classes for [SYCL\* Unified Shared Memory
```

(USM)](https://link.springer.com/chapter/10.1007/978-1-4842-5574-2_6)
management and implements a tensor [array
API](https://data-apis.org/array-api/latest/).

The library helps authors of Python native extensions written

> in C*, Cython*, or pybind11* to access `dpctl` objects representing SYCL*

Reviewer: ```suggestion
in C\*, Cython\*, or pybind11\* to access `dpctl` objects representing SYCL\*
```

Author: Good point about C* as an actual language!

devices, queues, memory, and tensors.

> `Dpctl` is the core part of a larger family of [data-parallel Python*

Reviewer: ```suggestion
`Dpctl` is the core part of a larger family of [data-parallel Python\*
```

libraries and tools](https://www.intel.com/content/www/us/en/developer/tools/oneapi/distribution-for-python.html)
to program on XPUs. 

# Installing

You can install the library with [conda*](https://anaconda.org/intel/dpctl) and

> [pip*](https://pypi.org/project/dpctl/). It is also available in the [Intel(R)
> Distribution for
> Python*](https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/distribution-for-python.html)
> (IDP).
> 

Reviewer: ```suggestion
[pip\*](https://pypi.org/project/dpctl/). It is also available in the [Intel(R)
Distribution for
Python\*](https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/distribution-for-python.html)
(IDP).

```

## Inte(R) oneAPI

You can find the most recent release of `dpctl` every quarter as part of the Intel(R) oneAPI releases. 

To get the library from the latest oneAPI release, follow the
instructions from Intel(R) [oneAPI installation
guide](https://www.intel.com/content/www/us/en/developer/articles/guide/installation-guide-for-oneapi-toolkits.html).

> > **NOTE:** You need to install the Intel(R) oneAPI Base Toolkit to get

Author: Should we change it to:
```suggestion
> **NOTE:** You need to install the Intel(R) AI Analytics Toolkit (AI Kit) to get
```

> > **NOTE:** You need to install the Intel(R) oneAPI Base Toolkit to get

Reviewer: We are in the process of transition from BaseKit to AI Kit, but that transition will only become public in 2023. Since AI kit build on BaseKit changing it now will not be misleading.

>IDP and `dpctl`.

## Conda

> To install `dpctl` from the Intel(R) channel on Annaconda*

Reviewer: ```suggestion
To install `dpctl` from the Intel(R) channel on Annaconda\*
```

cloud, use the following command:

```bash
conda install dpctl -c intel
```

## PyPi

> To install `dpctl` from PyPi*, run the following command:

Reviewer: ```suggestion
To install `dpctl` from PyPiz\*, run the following command:
```

Reviewer: LOL, fixed.

instructions from Intel(R) [oneAPI installation
guide](https://www.intel.com/content/www/us/en/developer/articles/guide/installation-guide-for-oneapi-toolkits.html).

> **NOTE:** You need to install the Intel(R) oneAPI Basekit to get
>IDP and `dpctl`.

## Conda

> To install `dpctl` from the Intel(R) channel on Annaconda

Reviewer: ```suggestion
To install `dpctl` from the Intel(R) channel on Anaconda
```

Cf. https://anaconda.org/

Reviewer: This was a pre-existing typo, but this PR seems appropriate set of changes to fix it.

Author: Good catch!

Installing the bleeding edge
------------------------

To try out the current master, install it from our

> development channel on Anaconda* cloud:

Reviewer: ```suggestion
development channel on Anaconda\* cloud:
```

```bash
conda install dpctl -c dppy\label\dev
```

# Building

Refer to our [Documentation](https://intelpython.github.io/dpctl) for more information on
setting up a development environment and building `dpctl` from the source.

# Running Examples

Find our examples [here](examples). 

To build these examples, run:

```bash
for script in `ls examples/python/`; 
    do echo "executing ${script}"; 
    python examples/python/${script}; 
done
```

> ##  Cython* extensions

Reviewer: ```suggestion
##  Cython\* extensions
```
